langchanges.rst

```diff
@@ -47,8 +47,8 @@ You will first need a PEP that you will present to python-ideas. You may be a
 little hazy on the technical details as various core developers can help with
 that, but do realize that if you do not present your idea to python-ideas or
 python-list ahead of time you may find out it is technically not possible
-(e.g., Python's parser will not support the grammar change as it is an LL(1)
-parser). Expect extensive comments on the PEP, some of which will be negative.
+(e.g., Python's parser will not support the grammar change as a PEG parser).
```

Comment:
I think the whole sentence in parentheses should be removed

Reply:
removed and updated PR